src/main/java/mate/academy/dao/jdbc/DriverDaoJdbc.java

```diff
@@ -0,0 +1,118 @@
+package mate.academy.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.DriverDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Driver;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoJdbc implements DriverDao {
```

Comment:
```suggestion
public class DriverDaoJdbcImpl implements DriverDao {
```
same for other JdbcImpl classes